crates/blockchain/error.rs

```diff
@@ -1,10 +1,8 @@
-use thiserror::Error;
-
 use ethereum_rust_core::types::InvalidBlockHeaderError;
 use ethereum_rust_storage::error::StoreError;
 use ethereum_rust_vm::EvmError;
 
-#[derive(Debug, Error)]
+#[derive(Debug, thiserror::Error)]
```

Comment:
Is this necessary?

Reply:
Ah, just made it that way to keep it homogeneous with the rest of our error modules

Reply:
(and also I like the information that the Error is from thiserror right in the declaration)